helpers/prepare.sh

```diff
@@ -199,7 +199,7 @@ prepare_binary_arr()
   # lets try to get an unique binary array
   # Necessary for providing BINARIES array (usable in every module)
   export BINARIES
-  readarray -t BINARIES < <( find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type f -executable -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 )
+  #readarray -t BINARIES < <( find "$FIRMWARE_PATH" "${EXCL_FIND[@]}" -type f -executable -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 )
```

Comment:
What is the problem with this line?

Reply:
Iin this find we are looking for executables according the permissions (x-permission). I have removed this and now we are using the backup search which is searching for ELF files without the need of x permission. As I had multiple firmwares with brocken permissions we should not rely on permissions

Reply:
Thank you 👍